integration-test-jdbc/src/test/kotlin/integration/jdbc/JdbcUpdateSingleTest.kt

```diff
@@ -179,4 +182,15 @@ class JdbcUpdateSingleTest(private val db: JdbcDatabase) {
             }.let { }
         }
     }
+
+    @Test
+    fun idColumnOnlyEntity() {
+        val a = Meta.idColumnOnlyAddress
+        val query = QueryDsl.from(a)
+        val address: List<IdColumnOnlyAddress> = db.runQuery { query }
+        assertThrows<IllegalArgumentException> {
+            val updateQuery = QueryDsl.update(a).batch(address)
```

Comment:
Since this file is JdbcUpdateSingleTest.kt, I think it would be appropriate to use the single function here instead of the batch function.

Reply:
Sorry, I fixed.